src/gpodder/schema.py

```diff
@@ -312,6 +338,9 @@ def convert_gpodder2_db(old_db, new_db):
                 '',
                 None,
                 None,
+                0,
+                0,
+                row['author'] or '',
```

Comment:
idem: no 'author'

Reply:
resolved in [998c007](https://github.com/gpodder/gpodder/pull/1432/commits/998c007d8e12dcd69f238ab0d00942034c234fdc)